nextmove/src/main/java/no/difi/meldingsutveksling/arkivmelding/ArkivmeldingUtil.java

```diff
@@ -1,38 +1,64 @@
 package no.difi.meldingsutveksling.arkivmelding;
 
-import com.google.common.collect.Lists;
-import no.arkivverket.standarder.noark5.arkivmelding.Arkivmelding;
-import no.arkivverket.standarder.noark5.arkivmelding.Dokumentbeskrivelse;
-import no.arkivverket.standarder.noark5.arkivmelding.Journalpost;
-import no.arkivverket.standarder.noark5.arkivmelding.Saksmappe;
+import no.arkivverket.standarder.noark5.arkivmelding.*;
+import no.difi.meldingsutveksling.domain.MeldingsUtvekslingRuntimeException;
+import org.eclipse.persistence.jaxb.JAXBContextFactory;
 
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import javax.xml.bind.Unmarshaller;
+import javax.xml.transform.stream.StreamSource;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.util.Comparator;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Map'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AUselessImportCheck)